python/python/res/job_queue/job_manager.py

```diff
@@ -336,6 +336,7 @@ def postMessage(self, job=None, extra_fields={}, url=None):
 
         payload = {"user": self.user,
                    "cwd": os.getcwd(),
+                   "application": "ert",
```

Comment:
I would prefer e.g. `"application": "ert_forward_model"`  -this is not actually the main `ert` process logging, but rather one of the forward model components.